src/.vuepress/styles/index.styl

```diff
@@ -32,3 +32,41 @@ body
 
 .sidebar
     font-size: unset
+
+
+/* Accessibility: clear, high-contrast focus indicator for all focusable elements
+   - Prefer :focus-visible to avoid showing when elements are clicked with a mouse
+   - Provide a :focus fallback for older browsers
+*/
+:focus-visible
+  outline 2px solid #000
+  outline-offset 2px
+  box-shadow none
+
+/* Fallback for browsers that don't support :focus-visible */
+:focus
+  outline 2px solid #000
+  outline-offset 2px
+  box-shadow none
+
+/* Hide focus when an element is focused by mouse in browsers with :focus-visible support */
+:focus:not(:focus-visible)
+  outline none
+  box-shadow none
+
+/* API spec / Swagger content: make focus more prominent inside the generated
+   API docs area (.swagger-container) so form fields and interactive controls
+   have a larger 4px/4px focus ring. */
```

Comment:
These don't seem to have landed: 
<img width="415" height="303" alt="image" src="https://github.com/user-attachments/assets/8ff6750b-0085-4a56-b3eb-a4f6571760e6" />

Swagger has some more specific selectors that we could overwrite: `.link` seems one of them, I wonder if there are more...

Reply:
```styl
.swagger-ui is:(button:focus-visible, select:focus-visible, .link:focus, .opblock-summary-control:focus)
```

I think this would get all of them.

Reply:
I couldn't quite get this to work, but adding `!important` to the rule on line 60 fixes this.